examples/Api.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import metadata from 'component-metadata!react-big-calendar/Calendar';
-import transform from 'lodash/object/transform';
+import { transform } from 'lodash/object';
 
```

Comment:
That's an awful lot of work to avoid writing `import transform from 'lodash/object/transform';` while also being less clear.